administration/tests/test_admin_profiles.py

```diff
@@ -22,27 +22,28 @@ def test_get_profiles_not_authorized(self):
 
     def test_get_profile_id_not_authorized(self):
         response = self.client.get(
-            path="/api/admin/profiles/" + str(self.user.id) + "/"
+            path="/api/admin/profiles/" + str(self.company.id) + "/"
         )
         self.assertEqual(status.HTTP_401_UNAUTHORIZED, response.status_code)
 
-    def test_get_profiles(self):
+    def test_get_profiles_structure_json(self):
         self.client.force_authenticate(self.user)
         response = self.client.get(path="/api/admin/profiles/")
+        auto_data_user = response.json()[0]["person"]
```

Comment:
What is [0]? It's unclear and difficult to understand. What will happen if we change response structure? And auto_data_user['name'] is difficult to read to. Tests have to be as simple and clear as possible. I like your first varient more.

Reply:
It works stably and does not affect the order of starting the generators of the model